README.md

```diff
@@ -246,10 +246,21 @@ Alternatively, if you only care about the version you're currently running, you
 
 # Deploying to PyPi
 
-1. Update the version number in `setup.py`
+1. Update the version number in `learnosity_sdk/_version.py`
```

Comment:
I wonder if this section is needed at all, actually. The release script should be doing all that.

Reply:
Yeah, all that is needed now is to run a `make dist-upload`

Reply:
Ah ok, so the other steps in this section (steps 2-5 in "Deploying to PyPi") will result in:

- The release script prompting the engineer to enter the new version number
- `learnosity_sdk/_version.py` getting updated by the release script and committed to the repo
- Creation of the version tag, and
- The new version being deployed to PyPi

Is that accurate?

Reply:
That's what the other steps were doing, yeah.

Now, the only thing that the release script doesn't do is upload to PyPI, but that's what `make dist-upload` does, after a version check to make sure it's a clean release.